### PR TITLE
update the version of sdk in LDS installer

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by gamerson** [comment]
jenkins retest

File: build/installers/liferay-developer-studio/pom.xml [hunk 1]
```diff
@@ -35,8 +35,8 @@
         <portal-version>7.0.10.2</portal-version>
         <portal-zip-name>liferay-dxp-digital-enterprise-tomcat-7.0-sp2-20170317165321327.zip</portal-zip-name>
```
Comment (gamerson) [inline build/installers/liferay-developer-studio/pom.xml]: Do we need to update the portal zip as well?

File: build/installers/liferay-developer-studio/pom.xml [hunk 2]
```diff
@@ -35,8 +35,8 @@
...
         <plugin-sdk-download-base>https://cdn.lfrs.sl/repository.liferay.com/nexus/content/groups/public/com/liferay/portal/com.liferay.portal.plugins.sdk</plugin-sdk-download-base>
-        <plugin-sdk-zip-name>com.liferay.portal.plugins.sdk-1.0.9-withdependencies.zip</plugin-sdk-zip-name>
-        <plugin-sdk-version>1.0.9</plugin-sdk-version>
+        <plugin-sdk-zip-name>com.liferay.portal.plugins.sdk-1.0.11-withdependencies.zip</plugin-sdk-zip-name>
+        <plugin-sdk-version>1.0.11</plugin-sdk-version>
```
Comment (gamerson) [inline build/installers/liferay-developer-studio/pom.xml]: Does the new version include some needed fixes?  Does it make it better for studio customers?

Comment (jtydhr88) [inline build/installers/liferay-developer-studio/pom.xml]: As our test, when using the latest 1.0.11, we no need to download anything for ivy, but the older one still need to download some ivy files

**Pull request closed without merging**